.idea/runConfigurations/RunCLion.xml

```diff
@@ -1,6 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="RunCLion" type="GradleRunConfiguration" factoryName="Gradle">
-    <log_file alias="idea.log" path="$PROJECT_DIR$/plugin/build/clion-sandbox/system/log/idea.log" />
+      <log_file alias="idea-201.log" path="$PROJECT_DIR$/plugin/build-201/clion-sandbox/system/log/idea.log" />
+      <log_file alias="idea-193.log" path="$PROJECT_DIR$/plugin/build-193/clion-sandbox/system/log/idea.log" />
```

Comment:
Unfortunately, I don't know a way how to provide single path for all platform versions because run configurations support ant patterns only for filename